ggshield/core/config/auth_config.py

```diff
@@ -71,33 +42,71 @@ def expired(self) -> bool:
         )
 
 
-@dataclass
-class AuthConfig(YAMLFileConfig):
+def prepare_auth_config_dict_for_parse(data: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
I'm lacking a bit of context here but any reason why we support multiple accounts in the yaml file and only map it to a single account ? 
Would it make more sense to only support a single account in the YAML or multiple accounts in the InstanceConfig but only use the first one for now ?